imagemounter/util.py

```diff
@@ -204,4 +213,20 @@ def force_clean(execute=True):
             except Exception:
                 pass
 
-    return commands
\ No newline at end of file
+    return commands
+
+
+def lookup_guid(guid, parser=None):
+    # VMFS Datastore
+    if guid == '2AE031AA-0F40-DB11-9590-000C2911D1B8':
+        return 'vmfs'
+    # VMKCore Diagnostic
+    elif guid == '8053279D-AD40-DB11-BF97-000C2911D1B8':
+        return 'vmkcore-diagnostics'
+    elif guid == '6A898CC3-1DD2-11B2-99A6-080020736631' or \
+         guid == 'C38C896A-D21D-B211-99A6-080020736631':
+        return 'zfs-member'
+    else:
+        if parser:
+            parser._debug('    GUID {} not yet supported'.format(guid))
+        return 'unknown'
```

Comment:
There are more complete list available on the internet, so we could add a lot more already as this code now a filesystem which is recognized but supported for mounting exists gracefully.